tests/contrib/dbapi_async/test_dbapi_async.py

```diff
@@ -95,6 +95,27 @@ async def test_fetchone_wrapped_is_called_and_returned(self):
         assert "__result__" == await traced_cursor.fetchone("arg_1", kwarg1="kwarg1")
         cursor.fetchone.assert_called_once_with("arg_1", kwarg1="kwarg1")
 
+    @mark_asyncio
+    async def test_cursor_async_connection(self):
+        """Checks whether connection can execute operations with async iteration."""
+
+        def method():
+            pass
+
+        async with TracedAsyncCursor(self.cursor, Pin("dbapi_service", tracer=self.tracer), {}) as cursor:
+            cursor._trace_method(method, "my_name", "my_resource", {"extra1": "value_extra1"}, False)
```

Comment:
Can we add a comment describing why this line is required. I am not sure if we need this in this test. This test should validate that `cursor.execute("""select 'one' as x""")` and `cursor.execute("""select 'blah'""")` generate spans and `async for row in cursor` uses a valid async iterator